app/main.py

```diff
@@ -1,8 +1,60 @@
+from typing import Union
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: Union[float, int],
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def can_we_wash_car(self, car: Car) -> bool:
+        """
+        returns True if station.clean_power > car.clean mark else False
+        """
+        return self.clean_power > car.clean_mark
```

Comment:
I think it' also must be True if self.clean_power = car.clean_mark

Reply:
CarWashingStation provides services for Cars for getting income. If we get clean_power == car.clean_mark, we get 0 income in a formula, while wasting resources on washing car. So I disagree on this.